AI_ML/OPENAI/DALLE_IMAGE_GENERATOR/DALLE_IMAGE_GENERATOR.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from flojoy import flojoy, Image
+import openai
+from PIL import Image as PilImage
+import requests
+from io import BytesIO
+import os
+
+
+@flojoy
+def DALLE_IMAGE_GENERATOR(
+    prompt: str,
+    width: int = 1024,
+    height: int = 1024,
+) -> Image:
+    """
+    The DALLE_IMAGE_GENERATOR node takes a prompt and generates an image
+    using OpenAI's DALL-E model.
+    The prompt should be a sentence describing the image you want to generate.
+    The image will be returned as a DataContainer with the type 'image'.
+
+    Parameters
+    ----------
+    prompt: string
+        A sentence describing the image you want to generate.
+    width: int
+        The width of the generated image.
+    height: int
+        The height of the generated image.
+    """
+    openai.api_key = os.environ.get("OPENAI_API_KEY")
+    result = openai.Image.create(prompt=prompt, n=1, size=f"{width}x{height}")
```

Comment:
@vinicvaz I think we should add two things here (and in the other nodes as well):
- a check for `OPENAI_API_KEY` is not None
- a re-try policy, if error is `429 - Rate limit reached for requests`  [ref](https://platform.openai.com/docs/guides/error-codes)

what do you think?